### ISSUE TITLE
Compiling on Windows

### ISSUE THREAD

**Issue by fasihrana** [prevoty/jni-rs#96]
I'm trying to compile my example [fasihrana/rustjni_example](https://github.com/fasihrana/rustjni_example) on windows and I'm getting the following error:

```error: failed to run custom build command for `jni v0.10.1`
process didn't exit successfully: `C:\Code\RUSTSTUFF\rustjni_example\target\debug\build\jni-6a065823e243d0f6\build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'Failed to find libjvm.so. Try setting JAVA_HOME', C:\Users\frana\.cargo\registry\src\github.com-1ecc6299db9ec823\jni-0.10.1\build.rs:19:21
note: Run with `RUST_BACKTRACE=1` for a backtrace.```

Why is it looking for `libjvm.so` on Windows and how do I fix that?

**Comment by fpoli** [comment]
The error message is wrong: on windows it looks for `jvm.dll` (see [here](https://github.com/prevoty/jni-rs/blob/master/build.rs#L30)).

Have you set `JAVA_HOME`? Do you have `jvm.dll` somewhere in it?

The build script looks for `jvm.dll` in all the subdirectories of the path specified in the `JAVA_HOME` environment variable.

**Comment by alexander-irbis** [comment]
You should add a directory with jvm.dll into the PATH environment variable.

**Comment by fasihrana** [comment]
I have specified JAVA_HOME and the jvm.dll is in %JAVA_HOME%\jre\bin\server.

**Comment by fasihrana** [comment]
OK, I reinstalled Rust from its offline installer. After that the program was correctly complaining about jvm.lib not found. Then it complained about not finding jvm.lib when linking. that was fixed by adding a build.rs file with just the following:

```
fn main(){
        if cfg!(target_os = "windows") {
                println!(r"cargo:rustc-link-search=c:\Path\to\jdk\lib\");
        }
}
```

So, now I'm getting errors when I execute the binary compiled complaining that the jvm.dll could not be found even though the directory is in Path.
